lib/LMSManagers/LMSFinanceManager.php

```diff
@@ -3720,6 +3720,9 @@ public function GetReceiptList(array $params)
                 case 'address':
                     $where = ' AND address ?LIKE? ' . $this->db->Escape('%' . $search . '%');
                     break;
+                case 'positions':
+                    $positionsfilter = ' AND description ?LIKE? ' . $this->db->Escape('%' . $search . '%');
```

Comment:
A dlaczego to jest zrobione inaczej niż wszystkie pozostałe, gdzie przygotowujemy zmienną $where?

Reply:
To jest inna tabela... warunki dałem w JOINie bezpośrednio. Jak mogłoby to wyglądać w tym miejscu w zapytaniu SQL?

Reply:
Notacja warunków oddzielnie jest bardziej przejrzysta i potem pozwoli na szybsze np. rozbicie filtrów na kilka jeśli będzie taka potrzeba.

Reply:
Jak mam to przerobić żeby było zaakceptowane?

Reply:
Tak, żeby było jak inne.

Reply:
Tak jest ok?